stdlib/private/StdlibUnittestFoundationExtras/StdlibUnittestFoundationExtras.swift

```diff
@@ -75,6 +75,8 @@ public func autoreleasepoolIfUnoptimizedReturnAutoreleased(
 #endif
 }
 
+@_inlineable
+@_versioned
```

Comment:
This function has no body, it should not be @_inlineable (I'll add a diagnostic for that)